box-browse-sdk/src/main/java/com/box/androidsdk/browse/activities/BoxBrowseActivity.java

```diff
@@ -226,29 +204,51 @@ private void clearSearch() {
     }
 
     @Override
-    public void onMoreResultsRequested(BoxRequestsSearch.Search searchRequest) {
-        clearSearch();
+    public void onItemClick(BoxItem item) {
+        // Notify fragment about item click
+        BoxBrowseFragment browseFrag = getTopBrowseFragment();
```

Comment:
The adapter is in the fragment I assume.  Anyways if there is any logic that the fragment wants to handle it should pass itself as a listener to the adapter and handle it without the activity knowing.